libsplinter/src/rest_api/errors.rs

```diff
@@ -49,6 +51,9 @@ impl fmt::Display for RestApiServerError {
             RestApiServerError::BindError(e) => write!(f, "Bind Error: {}", e),
             RestApiServerError::StartUpError(e) => write!(f, "Start-up Error: {}", e),
             RestApiServerError::StdError(e) => write!(f, "Std Error: {}", e),
+            RestApiServerError::InvalidConfig(field) => {
```

Comment:
This should probably use the `InvalidStateError` from `crate::error`

Reply:
Updated